packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -968,6 +968,30 @@ Specifies a particular platform or device to match on
 }
 ```
 
+#### Device Condition object
+
+<ApiLifecycle access="ie" /><br>
+
+> **Note:** This feature is only available as a part of the Okta Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+
+Specifies the device condition to match on
+
+| Parameter | Description              | Data Type | Required |
+| ---       | ---                      | ---       | ---      |
+| registered   | If the device is registered | Boolean     | No      |
+| managed   | If the device is managed | Boolean     | No      |
```

Comment:
Should we elaborate a bit more on what `registered` and `managed` means here?  Registered => Registered with okta or (registered with OV). Managed => device is managed by an MDM and it should have a link to the Device Management section here